ui/src/containers/VisualiseForm/StatementsForm/AxesEditor/CourseEditor.js

```diff
@@ -0,0 +1,62 @@
+import React, { Component, PropTypes } from 'react';
+import { Map } from 'immutable';
+import FullActivitiesList from 'ui/containers/FullActivitiesList';
+import {
+  getTypeOpts,
+  UNIQUENESS_OPS,
+  VALUE_OPTS,
+  UNIQUENESS_VALUE_OPTS,
+} from 'ui/utils/visualisations/localOptions';
+
+export default class CourseEditor extends Component {
+  static propTypes = {
+    operator: PropTypes.string,
+    value: PropTypes.instanceOf(Map),
+    model: PropTypes.instanceOf(Map),
+    updateModel: PropTypes.func.isRequired,
+  }
+
+  static defaultProps = {
+    operator: 'uniqueCount',
+    value: UNIQUENESS_VALUE_OPTS.get('statements'),
+  }
+
+  state = {
+    operatorInput: '',
+  }
+
+  changeOperator = newOperator =>
+    this.props.changeOperator(newOperator.keySeq().first())
+
+  includes = ys => xs =>
+    xs.indexOf(ys) !== -1
+
+  isUniquenessOp = op =>
+    this.includes(op)(UNIQUENESS_OPS)
```

Comment:
We can use *Array.prototype.includes()*
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/includes

```suggestion
    UNIQUENESS_OPS.includes(op)
```